.github/workflows/rc_sync.yml

```diff
@@ -71,3 +74,4 @@ jobs:
           pr_body: "Automatic sync from the release candidate to master during a feature freeze."
           pr_allow_empty: false
           pr_draft: false
+          pr_reviewer: "antalszava"
```

Comment:
@dime10 would you be in for being added too? Better to keep an eye on these PRs, especially while we're experimenting with them.